app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheet.kt

```diff
@@ -202,6 +207,15 @@ class ScheduleAppointmentSheet : BottomSheetActivity(), ScheduleAppointmentUi, S
     finish()
   }
 
+  override fun openTeleconsultStatusSheet(teleconsultRecordUuid: UUID) {
+    activity.startActivity(
```

Comment:
We would have to change this to close the `ScheduleAppointmentSheet` with activity result and intent extra to indicate the patient summary to open `TeleconsultStatusSheet`

Please refer to this [message](https://simpledotorg.slack.com/archives/CFK2PHJ1L/p1603855213202900) for more information

Reply:
We might have to make a different change here, we will keep open logic as it is, but instead use activity results to close this sheet and subsequently go to home screen from patient summary